src/bot/ui/printToConsole.js

```diff
@@ -216,7 +217,7 @@ function printToConsole({
 			ui.div(
 				{
 					text: `IN:  ${chalk.yellowBright(
-						toDecimal(route.inAmount, inputToken.decimals)
+						toDecimal(JSBI.toNumber(route.inAmount), inputToken.decimals)
```

Comment:
Converting to a string seems to be more reliable than converting to a number with the JSBI library.

`JSBI.toNumber(route.inAmount)` 

vs

`Number(route.inAmount.toString())`

![image](https://user-images.githubusercontent.com/16689258/210421856-82f67eab-7c10-4a80-9969-79d31ba680cc.png)